### DISCUSSION TITLE
architecture/devops

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#253] [General]
# architecture/devops

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/architecture/devops.html

**Comment by spikfaye** [comment]
如何打破开发和运维的之间的壁垒？
程序员：这样那样再这样
老板：开发和运维是同一批人不就好了

**Reply by isno** [reply]
这老板怪聪明的来

**Reply by JeraldZhang** [reply]
同一个人不更好？我就是同一个人😂

**Comment by XuankangLin** [comment]
> 这也解释了为何 DevOps 理念在十多年前就已提出，但直到近几年才开始被企业广泛关注和实践的原因。

句子开头的 “为何” 与最后的 “的原因” 语义重复了，可以删一个

**Reply by isno** [reply]
好的，谢谢。可以删除一个
